### PR TITLE
Wish list

### PR DESCRIPTION
**Pull request by sanathkumar26700**
Added wish list  page and functionalities like,

1. Add to wishlisht
2. Remove from wishlist
3. Add wishlist counter badge in navbar
4. notifications using toasts

### CODE REVIEW

File: src/Components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -61,7 +65,7 @@ function Navbar() {
                             <div className="nav-list--item__icon--wrapper">
                                 <span>
                                         <i className="fas icon fa-heart">
-                                            <span className="status-badge notification-badge">6</span>
+                                            <span className="status-badge notification-badge">{isAuthorized ? wishListData.length : 0}</span>
```
Comment (himadri2110) [inline src/Components/Navbar/Navbar.jsx]: Suggestion: You can choose not to show any badge if the user is not logged in. Not even 0.

File: src/Pages/Wish List/wish-list.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.page--heading {
+  width: 75%;
+  margin: auto;
+  font-size: 2.5rem;
+  padding: 2rem;
+  font-size: 4rem;
```
Comment (himadri2110) [inline src/Pages/Wish List/wish-list.css]: You can remove one `font-size` property

**Review by himadri2110 — APPROVED** [review]
Wishlist functionality works well.
Good to merge!

**Review by lomesshh — APPROVED** [review]
clean and modular code
UI looks good
all functionalities are working as expected
looks good to merge

**Pull request merged**